server/logging/gunicorn.py

```diff
@@ -126,3 +126,7 @@ def access(self, resp, req, environ, request_time):
 
         struct_pb = ParseDict(message, Struct())
         self.cloud_logger.log_proto(struct_pb, severity=severity)
+
+
+if os.getenv('APPLICATION_INSIGHTS_KEY'):
```

Comment:
Should this be `APPLICATION_INSIGHTS_KEY` or `APPINSIGHTS_INSTRUMENTATIONKEY`?

Reply:
Good catch. This should be `APPINSIGHTS_INSTRUMENTATIONKEY` as per https://github.com/Microsoft/ApplicationInsights-Python/pull/79. Fixed and also rebased latest master.